eng/pipelines/handlers.yml

```diff
@@ -106,6 +106,8 @@ parameters:
         tfm: default
       # - name: net6
       #   tfm: net6.0
```

Comment:
This is still disabled because of https://github.com/dotnet/runtime/issues/67660 and https://github.com/dotnet/linker/issues/2203